openstack/bms/v2/tags/create.go

```diff
@@ -0,0 +1,30 @@
+package tags
+
+import (
+	"github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/extract"
+)
+
+// CreateOpts contains all the values needed to create a new tag.
+type CreateOpts struct {
+	Tag []string `json:"tags" required:"true"`
+}
+
+// Create will create a new Tag based on the values in CreateOpts.
+func Create(c *golangsdk.ServiceClient, serverId string, opts CreateOpts) (*Tags, error) {
+	b, err := golangsdk.BuildRequestBody(opts, "")
+	if err != nil {
+		return nil, err
```

Comment:
You can pass `tags []string` argument to `Create` in this case, and return string slice too, if there is no additional information in request or response body

Reply:
BuildRequestBody cannot handle []string.